lib/src/widgets/content_item/content_menu.dart

```diff
@@ -168,6 +168,14 @@ void showContentMenu(
                     ),
                     onTap: () => showBookmarksMenu(context, widget),
                   ),
+                if (widget.onMarkAsRead != null)
+                  ListTile(
+                    title: Text(l(context).action_markRead),
```

Comment:
Could this text change based on whether the post is `read` or not? Like "Mark as read" if the post is unread, and "Mark as unread" if the post is read.